### PR TITLE
Kiwi 1665 qa

### PR DESCRIPTION
**Pull request by Z-Mohamoud**
### What changed

Add check for context field in CIC_CRI_START TxMA Event 

### Why did it change

New requirement added as part of KIWI-1665

### Issue tracking

- [KIWI-1665](https://govukverify.atlassian.net/browse/KIWI-1665)

### PII logging

- [ ] Verified that no PII data is being logged

### Environment variables or secrets

- [ ] No environment variables or secrets were added or changed

- [ ] Documented in the [README](./blob/main/README.md)
- [ ] Added to deployment repository
- [ ] Added to local startup repository

### Other considerations

- [ ] Update [README](./blob/main/README.md) with any new instructions or tasks

[KIWI-1665]: https://govukverify.atlassian.net/browse/KIWI-1665?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/tests/api/HappyPath.test.ts [hunk 1]
```diff
@@ -14,3 +14,16 @@ describe("/session Happy Path", () => {
 		await expect(getKeyFromSession(sessionId, constants.DEV_CIC_SESSION_TABLE_NAME, "journey")).resolves.toBe(journeyType);
 	});
 });
+
+describe("Happy Path - CIC_CRI_START Event Check", () => {
+	it.each([
+		["NO_PHOTO_ID", "bank_account"],
+	])("BAV and F2F test", async (journeyType: any, context: string) => {
```
Comment (Caitlin-cooling) [inline src/tests/api/HappyPath.test.ts:21]: [nit] `journeyType: string`

File: src/tests/utils/ApiTestSteps.ts [hunk 2]
```diff
@@ -294,3 +294,11 @@ export async function validateTxMAEventData(keyList: any): Promise<any> {
 			});
 	}
 }
+
+export async function validateCriStartTxMAEvent(key: any, context: string): Promise<any> {
+		const getObjectResponse = await HARNESS_API_INSTANCE.get("/object/" + key, {});
+		if (getObjectResponse.data.event_name === "CIC_CRI_START") {
+			console.log(JSON.stringify(getObjectResponse.data, null, 2));
+			expect(getObjectResponse.data.extensions.evidence.context).toBe(context);
```
Comment (Caitlin-cooling) [inline src/tests/utils/ApiTestSteps.ts:302]: We won't even have the `extension` object if the context isn't `bank_account` so I'm not totally sure how this is passing?

Comment (Caitlin-cooling) [reply src/tests/utils/ApiTestSteps.ts:302]: Sorry I'm confused though the new test added runs 
```
it.each([
		["NO_PHOTO_ID", "bank_account"],
	])
```

How is this passing for `NO_PHOTO_ID`?

Comment (Z-Mohamoud) [reply src/tests/utils/ApiTestSteps.ts:302]: A journey type of NO_PHOTO_ID sets the context to bank_account when making the call to start the stub 

image

Comment (Caitlin-cooling) [reply src/tests/utils/ApiTestSteps.ts:302]: oh! So we are doing bank account twice here?

Comment (Z-Mohamoud) [reply src/tests/utils/ApiTestSteps.ts]: Updates made to change test data from array to object

**Pull request merged**